source/applications/advanced/hand_eye_calibration/ur_hand_eye_calibration/universal_robots_perform_hand_eye_calibration.py

```diff
@@ -92,15 +92,15 @@ def _initialize_robot_sync(host: str):
 
     # To ensure that the application is compatable with further versions of UR controller
     if not con.negotiate_protocol_version():
-        raise RuntimeError(f"Protocol do not match")
+        raise RuntimeError("Protocol do not match")
```

Comment:
I think Trym wants us to use f" when printing. That is why I had it there in the first place (even though it makes no sense having it there...)

Reply:
Lint complains when you don't add a variable to it. @trym-b comments?

Reply:
it doesn't make sense to use f-string without arguments (I agree with the linter, and I've also made this mistake myself)

Reply:
btw, I thought I fixed all these in the `fix pylint warnings` PR. Are you missing a rebase?

Reply:
We also fixed this when pylint was updated. No need to have f-string if there is noe variable to insert